app/views/loans/index.html.erb

```diff
@@ -1,43 +1,159 @@
-<p id="notice"><%= notice %></p>
-
-<h1>Loans</h1>
-
-<table>
-  <thead>
-    <tr>
-      <th>Current balance</th>
-      <th>Original amount</th>
-      <th>Interest rate</th>
-      <th>Periods in year</th>
-      <th>User</th>
-      <th>Loan name</th>
-      <th>Monthly min payment</th>
-      <th>Schedule hash</th>
-      <th>Adj schedule hash</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <% @loans.each do |loan| %>
+<div class="row mb-3">
+  <div class="col-md-12">
+    <h1>
+      All loans
+    </h1>
+
+    <a href="/loans/new" class="btn btn-block btn-outline-secondary">
+      Add a new loan
+    </a>
+  </div>
+</div>
+
+
+<hr>
+
+<div class="row mb-3">
+  <div class="col-md-12">
+    <table class="table">
       <tr>
-        <td><%= loan.current_balance %></td>
-        <td><%= loan.original_amount %></td>
-        <td><%= loan.interest_rate %></td>
-        <td><%= loan.periods_in_year %></td>
-        <td><%= loan.user_id %></td>
-        <td><%= loan.loan_name %></td>
-        <td><%= loan.monthly_min_payment %></td>
-        <td><%= loan.schedule_hash %></td>
-        <td><%= loan.adj_schedule_hash %></td>
-        <td><%= link_to 'Show', loan %></td>
-        <td><%= link_to 'Edit', edit_loan_path(loan) %></td>
-        <td><%= link_to 'Destroy', loan, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+
+        <th>
+          Loan Name
+        </th>
+
+        <th>
+          Current Balance
+        </th>
+        
+        <th>
+          Original Amount
+        </th>
+
+        <th>
+          Interest Rate
+        </th>
+
+        <th>
+          Monthly Minimum
+        </th>
+        
+        <th>
+           Original Payoff Period
+        </th>
+
+        <th>
+           Adjusted Payoff Period
+        </th>        
+        
+        <th>
+        </th>
       </tr>
-    <% end %>
-  </tbody>
-</table>
 
-<br>
+      <% @loans.each_with_index do |loan , index| %>
+      <tr>
+
+        <td>
+          <%= loan.loan_name %>
+        </td>
+
+        <td>
+          <%= number_to_currency(loan.current_balance) %>
+        </td>
+        
+        <td>
+          <%= number_to_currency(loan.original_amount) %>
+        </td>
+
+        <td>
+          <%= number_to_percentage(loan.interest_rate) %>
+        </td>
+
+        <td>
+          <%= number_to_currency(loan.monthly_min_payment) %>
+        </td>
+        
+        <td>
+          <%= @pay_off_period[index] %>
+        </td>
+
+        <td>
+          <%= @pay_off_period_adj[index] %>
+        </td>
 
-<%= link_to 'New Loan', new_loan_path %>
+        <td>
+          <a href="/loans/<%= loan.id %>">
+            Show details
+          </a>
+        </td>
+      </tr>
```

Comment:
If you're set on keeping a table row for Loan, you could make this a view partial.

Reply:
Thanks, moved this to a partial.  This may or may not stay in the long run, but this will trouble shoot if my calculations are working correctly as I as more features in the future.